### DISCUSSION TITLE
Face recognition

### DISCUSSION THREAD

**Discussion by mrbluebrett** [tsightler/ring-mqtt#454] [Feature Requests]
Any chance to integrate face recognition into this?
I have seen this project:
https://github.com/dude123studios/SmarterRingV2
That does the thing in python. It shouldn’t be that hard to integrate…

**Comment by tsightler** [comment]
Not something I'm interested in adding but it's open source so you can fork it and add whatever you want since it "shouldn't be that hard integrate".  Note however that the ring-mqtt project does not use Python at all.

That being said, I think it would be pretty easy to integrate something like double-take with ring-mqtt although I'm not sure if the snapshot images from Ring cameras are high enough resolution, but if so you could easily create an automation that would send the snapshot image on motion to double-take and then another one that listens for results on the MQTT topic.

**Comment by mrbluebrett** [comment]
Thanks for your idea. I'm trying on this way. One question is the snapshot size coming this small from the ring doorbell itself?

**Reply by tsightler** [reply]
Yes, ring-mqtt does no manipulation of the snapshot image.  I think they are only 360p as they are primarily designed for notifications and thumbnails in the app and this lowers power usage for battery cameras and bandwidth and storage usage for Ring since everything has to go via their cloud.
